blocks/api/test/query.js

```diff
@@ -32,4 +32,21 @@ describe( 'query', () => {
 			expect( wp.element.renderToString( match ) ).to.equal( html );
 		} );
 	} );
+
+	describe( 'node()', () => {
+		it( 'should return a matcher function', () => {
+			const matcher = query.node();
+
+			expect( matcher ).to.be.a( 'function' );
+		} );
+
+		it( 'should return HTML equivalent WPElement of matched element', () => {
+			// Assumption here is that we can cleanly convert back and forth
+			// between a string and WPElement representation
+			const html = '<blockquote><p>A delicious sundae dessert</p></blockquote>';
+			const match = parse( html, query.node() );
+
+			expect( wp.element.renderToString( match ) ).to.equal( `<body>${ html }</body>` );
```

Comment:
The `body` comes from hpq assigning `innerHTML` of a new document on which to perform the querying:

https://github.com/aduth/hpq/blob/a5bda80/src/index.js#L22

I seem to recall there may be alternatives to consider here to be able to avoid it.